wikimedia/wikipedia-ios
# T381332

**Phabricator:** 
https://phabricator.wikimedia.org/T381332

### Notes
* Has PR updates from https://github.com/wikimedia/wikipedia-ios/pull/5113

### Test Steps
1. In the proper region and language, with a logged in user with personalized slides
2. Load 6+ articles into history then background and foreground
3. Navigate to profile and then year in review
4. Compare images/gifs and copy to Figma, ensure correct.
5. Repeat with logged out user, English user, and non English.
6. Check feature announcement while repeating with logged out user, ensure it is updated.

### Screenshots/Videos

https://github.com/user-attachments/assets/124b8973-4e9a-4997-a0be-a2f814532b9b
